templates/admin/save_results.html.twig

```diff
@@ -0,0 +1,87 @@
+{% extends '@EasyAdmin/default/layout.html.twig' %}
+{% block content %}
+    <link rel="stylesheet" href="{{ asset('build/tournament.css') }}">
+
+
+    <div class="row list_of_participant">
+        <div class="col-md-2"></div>
+        <div class="col-md-10">
+            <table>
+                <tr>
+                    <td>Місце</td>
```

Comment:
Translation is needed, probably it's not related to round robin system, but please implement trans if possible

Reply:
done